.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+tasks:
+  - init: docker-compose pull &&
+      docker-compose build &&
+      docker run --rm --interactive --tty --volume $PWD:/app composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+    command: |
+      docker-compose up -d
+
+ports:
+  - port: 8080
+    onOpen: ignore
+    visibility: public
+  - port: 443
+    onOpen: ignore
+    visibility: public
+  - port: 3001
+    onOpen: ignore
+    visibility: public
```

Comment:
```suggestion
```

HTTPS is taken care by Gitpod, so we don't need port 443. It would not work anyway, as it would be <1024.

Why do we need port 3001? Is this a leftover?

Reply:
Yep, its leftover. I'll remove it